src/data/data1d/filter1d/autoPhaseCorrection.js

```diff
@@ -12,17 +12,17 @@ export const name = 'Automatic phase correction';
  */
 
 const defaultOptions = {
-  magnitudeMode: false,
-  minRegSize: 30,
-  maxDistanceToJoin: 256,
+  minRegSize: 10,
+  maxDistanceToJoin: 128,
```

Comment:
Should this option not be dependant from the number of points in the spectrum ? We have spectra with 1k points and some with 256k points.

Reply:
It should be tested because I believe it does not relate with the resolution of the spectrum, it is closer to how much noise has the spectrum, but I put a bigger cutoff for noise, I think we could test with those parameters to check if it works for most cases.